fdbserver/DataDistribution.actor.cpp

```diff
@@ -1897,7 +1905,7 @@ ACTOR Future<bool> scheduleBulkDumpTasks(Reference<DataDistributor> self) {
 					if (!self->ongoingBulkDumpActors.liveActorAt(taskRange.begin)) {
 						// Limit parallelism
 						loop {
-							if (self->bulkDumpParallelismLimitor.tryIncrementTaskCounter()) {
+							if (self->bulkDumpParallelismLimitor.canStart()) {
```

Comment:
Is this right? Should it be not ?

Reply:
The blobstore uses FlowLock to limit parallelism. You want to do same?

Reply:
For the first comment, we want to break the loop when we can start so it is correct as is. And for the second comment, FlowLock wants to spawn the actor so is heavy weight for what is going on here.